.github/workflows/application.yml

```diff
@@ -791,6 +791,15 @@ jobs:
                     name: "Behat logs (non-JS, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
                     path: etc/build/
                     if-no-files-found: ignore
+            
+            -
+                name: Upload logs
+                uses: actions/upload-artifact@v3
+                if: failure()
+                with:
+                    name: "Logs (non-JS, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
+                    path: var/log/
+                    if-no-files-found: ignore
```

Comment:
:+1: but not related

Reply:
It was only for debugging purposes, I removed it c:.